### PR TITLE
Update Cluster Upgrades RFC

### PR DESCRIPTION
**Pull request by pipo02mix**
Towards https://github.com/giantswarm/giantswarm/issues/29973

### CODE REVIEW

File: automatic-cluster-upgrades/README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 ---
 creation_date: 2021-09-17
+issues:
+- [https:/](https://github.com/giantswarm/giantswarm/issues/27420)
+last_review_date: 2024-02-23
+owners:
+- https://github.com/orgs/giantswarm/teams/team-horizon
```
Comment (pipo02mix) [inline automatic-cluster-upgrades/README.md:7]:
```suggestion
owners:
- https://github.com/orgs/giantswarm/teams/team-horizon
- https://github.com/orgs/giantswarm/teams/team-honeybadger
```

File: automatic-cluster-upgrades/README.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
 ---
 creation_date: 2021-09-17
+issues:
+- [https:/](https://github.com/giantswarm/giantswarm/issues/27420)
```
Comment (JosephSalisbury) [inline automatic-cluster-upgrades/README.md:4]:
```suggestion
- https://github.com/giantswarm/giantswarm/issues/27420
```

File: automatic-cluster-upgrades/README.md [hunk 3]
```diff
@@ -1,6 +1,13 @@
...
+last_review_date: 2024-02-23
+owners:
+- https://github.com/orgs/giantswarm/teams/team-horizon
+- https://github.com/orgs/giantswarm/teams/team-honeybadger
 state: approved
+summary: As part of the Cluster API hackathon the idea of how to automate the cluster upgrades allows us to brainstorm how it can be achieved taking into account customer requirements and technical limitations brought under the new upstream implementation.
```
Comment (JosephSalisbury) [inline automatic-cluster-upgrades/README.md:10]: > how to automate the cluster upgrades allows us to brainstorm how

i don't know what you're trying to say here - we brainstormed how to automated cluster upgrades?

Comment (pipo02mix) [reply automatic-cluster-upgrades/README.md:10]: wow grammarly played on me. Yeah it was the intent

File: automatic-cluster-upgrades/README.md [hunk 4]
```diff
@@ -1,6 +1,13 @@
 ---
 creation_date: 2021-09-17
+issues:
+- https://github.com/giantswarm/giantswarm/issues/27420
+last_review_date: 2024-02-23
+owners:
+- https://github.com/orgs/giantswarm/teams/team-horizon
+- https://github.com/orgs/giantswarm/teams/team-honeybadger
 state: approved
+summary: As part of the Cluster API hackathon the idea of how to automate the cluster upgrades allows us to brainstorm how it can be achieved taking into account customer requirements and technical limitations brought under the new upstream implementation.
```
Comment (pipo02mix) [inline automatic-cluster-upgrades/README.md:10]:
```suggestion
summary: As part of the Cluster API hackathon we brainstormed how to automate the cluster upgrades taking into account customer requirements and technical limitations brought under the new upstream implementation.
```

**Pull request merged**